From dbc92d85f207f8182856b6c1a4a4230d109a8529 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 12 Oct 2012 11:56:07 +0200
Subject: [PATCH] nfs/install: don't install nss3.so

nss3.so has nothing to do with the desired nss_*.so files
---
 modules.d/95nfs/install | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/95nfs/install b/modules.d/95nfs/install
index 21f7ab6..5432a22 100755
--- a/modules.d/95nfs/install
+++ b/modules.d/95nfs/install
@@ -24,7 +24,7 @@ nsslibs=$(sed -e '/^#/d' -e 's/^.*://' -e 's/\[NOTFOUND=return\]//' /etc/nsswitc
 nsslibs=${nsslibs#|}
 nsslibs=${nsslibs%|}
 
-dracut_install $(for i in $(ls {/usr,}$LIBDIR/libnss*.so 2>/dev/null); do echo $i;done | egrep "$nsslibs")
+dracut_install $(for i in $(ls {/usr,}$LIBDIR/libnss_*.so 2>/dev/null); do echo $i;done | egrep "$nsslibs")
 
 inst_hook cmdline 90 "$moddir/parse-nfsroot.sh"
 inst_hook pre-pivot 99 "$moddir/nfsroot-cleanup.sh"
